src/sinol_make/commands/inwer/__init__.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+import os
+import multiprocessing as mp
+
+from sinol_make import util
+from sinol_make.commands.inwer.structs import TestResult, InwerExecution, VerificationResult, TableData
+from sinol_make.helpers import package_util
+from sinol_make.helpers.parsers import add_compilation_arguments
+from sinol_make.interfaces.BaseCommand import BaseCommand
+from sinol_make.commands.inwer import inwer_util
+
+
+class Command(BaseCommand):
+    """
+    Class for "inwer" command.
+    """
+
+    def get_name(self):
+        return "inwer"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Verify if input files are correct',
+            description='Verify if input files are correct'
+        )
+
+        parser.add_argument('inwer', type=str, nargs='?',
+                            help='path to inwer executable')
+        parser.add_argument('--tests', type=str, nargs='+',
+                            help='test to verify, for example in/abc{0,1}*')
+        parser.add_argument('--cpus', type=int,
+                            help=f'number of cpus to use, you have {mp.cpu_count()} available')
+        add_compilation_arguments(parser)
+
+    def compile_inwer(self, args: argparse.Namespace):
+        self.inwer_executable, compile_log = inwer_util.compile_inwer(self.inwer, args)
+        if self.inwer_executable is None:
+            print(util.error('Compilation failed.'))
+            os.system("head -c 500 %s" % compile_log)
+            exit(1)
+        else:
+            print(util.info('Compilation successful.'))
```

Comment:
This seems like a function that will also be used in `ingen` etc. Maybe move it to a helper module?

Reply:
Is there a Python `head` alternative? Using commands from shells/operating systems can have unexpected output or behavior on some shells/operating systems. To have a guaranteed stability, it's preferable to use language features or modules written by other people that are sure to work on each operating system.

Reply:
I added a function for printing first 500 lines from compile log e94fdfd5705b3701f57f578106d9c15200359b61.